### ISSUE TITLE
How to manage  the tmp folders?

### ISSUE THREAD

**Issue by PeterWillems** [sysunite/coins-2-sdk#2]
The COINS-API software installs and uses temporary folders to store session data. The storage size of these folders increases over time because the tools themselves do not remove obsolete folders.
Since I am developing a browsing/editing end user tool that makes use of the COINS-API functionality as a kind of middle-ware I am curious how I can maintain these temporary folders: e.g. I want to be able to remove obsolete files and use another folder directory (by the way C:\tmp could orginally not be created on my company PC, happily I have administrator rights, but you will understand the issue).
So, which (hidden) functions are available for me to control this part of the COINS-API?

**Comment by bastbijl** [comment]
The FileManager class manages the use of temporary files.

By setting these the default location can be changed:
```java
FileManager.TEMP_ZIP_PATH = "" // default: /tmp/coinscontainers
FileManager.LIB_FOLDER = "" // default: /tmp/coinsgloballibraries
```

To cleanup the used temporary files this command must be called:

```java
FileManager.destroyAll()
```

**Comment by PeterWillems** [comment]
Exactly what I was looking for, thanks.

**Comment by PeterWillems** [comment]
The FileManager.destroyAll call only seems to clean up the temporary ZIP folder. However, the temporary LIB folder is not affected. Is this a feature or a bug?

**Comment by bastbijl** [comment]
A bug! This will be solved in the 1.1 version of the API. It might take some time before this version is released.
